.github/workflows/tests.yml

```diff
@@ -43,4 +40,4 @@ jobs:
         with:
           go-version: 1.17
       - run: |
-          go test -v --tags "${{ matrix.tags }}" ./...
+          go test -v ./...
```

Comment:
@d-bytebase then you don't test with `--tags slim` but assume that it should be OK...